src/olympia/addons/cron.py

```diff
@@ -39,6 +39,17 @@ def update_addon_average_daily_users():
     kwargs = {'id_field': 'pk'}
     if waffle.switch_is_active('use-bigquery-for-addon-adu'):
         d = get_addons_and_average_daily_users_from_bigquery()
+        # BigQuery does not have data for add-ons with type other than those in
+        # `ADDON_TYPES_WITH_STATS` so we use download counts instead.
+        # See: https://github.com/mozilla/addons-server/issues/14609
+        d += (
+            Addon.objects
+                .exclude(type__in=amo.ADDON_TYPES_WITH_STATS)
+                .annotate(count=Sum('downloadcount__count'))
+                .values_list('guid', 'count')
+                .order_by('guid')
+                .all()
```

Comment:
@diox I don't like the fact that we retrieve all results first but I don't have a nicer way. WDYT?

Reply:
That's probably fine, but I'd do the `Addon.objects.[...]` call first. The reason is, you might lose the connection to the database if the big query call takes too much time, so it'd be safer if the 2 were swapped.